osmaxx/conversion/converters/converter_gis/gis.py

```diff
@@ -32,22 +31,19 @@ def __init__(self, *, conversion_format, out_zip_file_path, base_file_name, out_
         self._static_directory = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'static')
 
     def create_gis_export(self):
-        with pbf_area(osmosis_polygon_file_content=self._polyfile_string) as pbf_path:
```

Comment:
This was the only production code using the `pbf_area` CM. The test code seems to access it to test the CM, not as a tool for testing other stuff. So the CM implementation could now be removed as well.